src/sys/socket/ffi.rs

```diff
@@ -20,7 +20,10 @@ pub type type_of_cmsg_len = socklen_t;
 #[cfg(target_os = "macos")]
 pub type type_of_cmsg_data = c_uint;
 
-#[cfg(not(target_os = "macos"))]
+#[cfg(target_os = "dragonfly")]
```

Comment:
No, this is correct! ```type_of_cmsg_data``` is already defined above as c_uint. Here it is defined for DragonFly as c_int. Below it is defined for all other OSes.